content/docs/cucumber/configuration.md

```diff
@@ -231,7 +231,48 @@ This is just a convention though; Cucumber will pick them up from any file{{% te
 # Profiles
 
 {{% block "java" %}}
-Profiles are not available in Java.
+Profiles are not available in Java.  However, it is possible to set configuration options using [Maven profiles](https://maven.apache.org/guides/introduction/introduction-to-profiles.html).
+
+For instance, we can configure separate profiles for scenarios which are to be run in separate environments like so:
+
+``` xml
+    <profiles>
+        <profile>
+          <id>dev</id>
+            <properties>
+                <cucumber.options>--tags @dev</cucumber.options>
+                <base.url>http://dev.base.url.to.application</base.url>
```

Comment:
Is the base url needed in this example?

Reply:
It would be if the profile is used to set different environment variables. In which case the tag example above would not be needed. Either one or the other would be sufficient.

Reply:
I've removed it from the example :)